demo/play-demo-for-recording.sh

```diff
@@ -0,0 +1,208 @@
+#!/bin/bash
+
+#asciinema rec -w 2
+
+exec() {
+  echo "\$ $@"|pv -qL 10 ; "$@" ;
+  }
+
+echo "   "
+echo "   "
+echo " Simplify the deployment of Spring Boot applications using Halkyon Component Operator on Kubernetes"|pv -qL 10
+echo " In this demo, we will :"|pv -qL 10
+echo "   --> Compose & link 2 microservices: client and fruits backend"|pv -qL 10
+echo "   --> Deploy a capability such as a database and link it to backend microservice accessing it"|pv -qL 10
+echo "   --> Code locally and next push/build on Kubernetes/OpenShift"|pv -qL 10
+echo "  "|pv -qL 10
+echo " Ready? Let's begin!"|pv -qL 10
+echo "  "
+echo "  "
+sleep 5
+
+clear && sleep 1
+echo "# Log on to the cluster using the oc client"|pv -qL 10
+sleep 2
+exec oc login https://159.69.209.188:8443 --token=5tKSUvACZ8big9XT2mONlqMU5J4w6Di6RFe9wrnJiU0
+sleep 3
+
+clear && sleep 1
+echo "# Create a new project"|pv -qL 10
+sleep 1
+exec oc new-project demo
+sleep 7
+
+clear && sleep 1
+echo "# Create a directory named demo and subsequently navigate to it"|pv -qL 10
+sleep 2
+exec mkdir demo
+sleep 1
+exec cd demo
+sleep 3
+
+clear && sleep 1
+echo "# Create a pom.xml with the following content"|pv -qL 10
```

Comment:
`with the following content` --> `with 2 modules corresponding each to a microservice`